atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +135,13 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        for platform in worker_metadatas:
+            annotations = get_worker_build_info(self.workflow, platform).build.get_annotations()
+            if 'media_version' in annotations and annotations['media_version']:
+                extra['image']['media_types'] = annotations['media_version']
```

Comment:
Any reason not to call the annotation "media-types"? ('types' for consistency with the CG metadata name, and '-' for consistency with other annotations)